scripts/batch-score.coffee

```diff
@@ -0,0 +1,107 @@
+# Description:
+#   Script for batch wise score.
+#
+# Configuration:
+#   INFO_SPREADSHEET_URL
+#
+# Commands:
+#   hubot score bxx
+# 
+# Author:
+#   aseem09
+
+module.exports = (robot) ->
+
+  # using this, stores the data in brain's "scorefield" key
```

Comment:
drop "using this": `get the data of brain's scorefield key` is just fine. I think we can drop the comment altogether.
Tip: write code that speaks for itself, i.e., that need not be commented.